packages/@uppy/audio/src/DiscardButton.js

```diff
@@ -0,0 +1,31 @@
+const { h } = require('preact')
+
+function DiscardButton ({ onDiscard, i18n }) {
+  return (
+    <button
+      className="uppy-u-reset uppy-c-btn uppy-Audio-button"
+      type="button"
+      title={i18n('discardRecordedFile')}
+      aria-label={i18n('discardRecordedFile')}
+      onClick={onDiscard}
+      data-uppy-super-focusable
+    >
+      <svg
+        width="13"
+        height="13"
+        viewBox="0 0 13 13"
+        xmlns="http://www.w3.org/2000/svg"
+        aria-hidden="true"
+        focusable="false"
```

Comment:
AFAIK this attribute is not a valid `<svg>` attribute.
EDIT: apparently it was valid under an older version of SVG spec: https://stackoverflow.com/a/28654521/6549112. I don't think we need to keep it.

```suggestion
```

Reply:
It was needed for some old IE, I think, good to remove now!